src/couch/src/couch_util.erl

```diff
@@ -62,6 +63,37 @@ normparts(["." | RestParts], Acc) ->
 normparts([Part | RestParts], Acc) ->
     normparts(RestParts, [Part | Acc]).
 
+
```

Comment:
Answering my own question:

This implementation doesn't do useless call to `file:read_file_info(File)`.